db/migrate/20170406140542_add_ratings_association_to_restaurant.rb

```diff
@@ -0,0 +1,5 @@
+class AddRatingsAssociationToRestaurant < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :ratings, :restaurant, foreign_key: true
```

Comment:
yes but i think that it's god to make a separate PR for that and add a user page that list old orders and asks the user to rate used restaurants.